app/models/admin/proposals/proposal.rb

```diff
@@ -30,7 +30,14 @@ class Admin::Proposals::Proposal < ApplicationRecord
   include LabelHelper
   has_paper_trail
 
-  validates :show_title, :proposal_text, :publicity_text, :call_id, presence: true
+  validates :show_title, :proposal_text, :publicity_text, :call_id, :status, presence: true
+
+  enum status: {
+    awaiting_approval: 0,
+    approved: 1,
+    unsuccessful: 2,
```

Comment:
Does unsuccessful mean not approved and failed mean lost the vote? Maybe worth making that clearer (also is the reason for there not being a succeeded value just that it will be converted from a proposal to a show?)

Reply:
I think I included this too soon. It's still WIP, but yes, I should rethink this. I can't remember why I put those four options and not a successful one.